about.html

```diff
@@ -0,0 +1,182 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About Page</title>
+    <!-- bootstrap icons -->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css">
+
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+
+    <link
+    href="https://cdnjs.cloudflare.com/ajax/libs/mdb-ui-kit/5.0.0/mdb.min.css"
+    rel="stylesheet"
+  />
+  <!-- Font Awesome -->
+    <link
+    href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css"
+    rel="stylesheet"
+    />
+    <!-- Google Fonts -->
+    <link
+    href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap"
+    rel="stylesheet"/>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+        <!-- Media header  -->
+
+        <nav class="navbar d-flex justify-content-end navbar-dark bg-dark">
+            
+            <a class="navbar-brand mb-0 h1"><i class="bi bi-facebook"></i></a>
+            <a class="navbar-brand mb-0 h1"><i class="bi bi-twitter"></i></a>
+            <a class="navbar-brand mb-0 h1">English</a>
+            <a class="navbar-brand mb-0 h1">My Page</a>
+            <a class="navbar-brand mb-0 h1">Logout</a>
+            
+        </nav>
```

Comment:
- [ ] According to the requirements and the original design, You should follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including:
 Colors. So kindly the change `background-color` for the Top Navbar to `dark/black`  👍

| Your Design  | Original Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/198771964-57499148-bc99-466b-a739-5207162d6a7c.PNG" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/198772056-2cd6abfe-d769-4916-be46-ea58e60028c9.PNG" width="500" /> |

- [ ] Also, this navbar should disappear on Mobile Version, I suggest to fix it 👍

 | Your Design  | Original Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/198773815-41306f74-4647-47e0-ac02-218db115ff11.png" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/198773385-66c72c35-1ee5-4af6-aab7-191759df4a18.png" /> |


> Please, apply the Same thing on both About and Home pages 💯